### ISSUE TITLE
Create a Contributor's Quick Start Guide

### ISSUE THREAD

**Issue by RNHTTR** [MarquezProject/marquez#2006]
It'd be helpful for new contributors to have a quick start guide to help them get up-to-speed quickly. This could potentially be added to CONTRIBUTING.md, but ultimately I think something should touch on how to use `up.sh` locally for development, e.g. `docker/up.sh --seed --build`

**Comment by wslulciuc** [comment]
@RNHTTR: we do have a [quickstart](https://marquezproject.ai/quickstart) for Marquez, but did you have something else in mind?

**Comment by RNHTTR** [comment]
Yeah, specifically for contributing to Marquez. Maybe a section in CONTRIBUTING.md that touches on the `--build` flag to build a local image from the developer's current branch
